### ISSUE TITLE
b.functions.source is undefined

### ISSUE THREAD

**Issue by conrad66** [artem-sedykh/mini-climate-card#1]
Does not work
![Screenshot_6](https://user-images.githubusercontent.com/5843698/83670404-c7661b80-a5db-11ea-98af-cb19c414996e.png)

**Comment by ktownsend-personal** [comment]
I'm getting related error in Chrome, which I traced to b.functions.source being undefined:
```
Cannot set property '__init' of undefined
type: 'custom:mini-climate'
entity: climate.upstairs
```

**Comment by deco3690** [comment]
![image](https://user-images.githubusercontent.com/26707592/83738615-1cab3700-a687-11ea-8197-f557adb40b47.png)

**Comment by artem-sedykh** [comment]
fixed in [v1.0.2](https://github.com/artem-sedykh/mini-climate-card/releases/tag/v1.0.2)
